config/environment.js

```diff
@@ -144,6 +144,11 @@ module.exports = function(environment) {
             popularNode,
             noteworthyNode,
         },
+        featureFlags: { // default flags (whether they be switches, flags, or polls) go here with default value.
+            ember_support_page: true,
+            ember_dashboard_page: true,
+            _loaded: false,
```

Comment:
I think this makes more sense as a property on `current-user`, since that's the only place that uses it.